extensions/azure/cosmos/policy-store-cosmos/build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+/*
+ *  Copyright (c) 2020, 2021 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+plugins {
+    `java-library`
+}
+
+val cosmosSdkVersion: String by project
+val jodahFailsafeVersion: String by project
+
+dependencies {
+    api(project(":spi:policy-spi"))
+    api(project(":common:util"))
```

Comment:
for what I understood, `implementation` should be used, as we don't want to expose these